ddtrace/contrib/internal/pymongo/patch.py

```diff
@@ -60,43 +73,39 @@ def unpatch():
 
 def patch_pymongo_module():
     _w(pymongo.MongoClient.__init__, _trace_mongo_client_init)
-    _w(pymongo.topology.Topology.select_server, _trace_topology_select_server)
+    _w(Topology.select_server, _trace_topology_select_server)
```

Comment:
For testing do we just want to assert that we've successfully wrapped these methods?

Reply:
Yup